docs/TROUBLESHOOTING.md

```diff
@@ -214,13 +214,13 @@ This section includes how to debug a failed restore. For more specific issues re
 
 <hr style="height:1px;border:none;color:#333;"> 
 
-<h3 align="center">Issue with Backup/Restore of DeploymentConfig using Restic or restore hooks<a id="deployconfig"></a></h3>
+<h3 align="center">Issue with Backup/Restore of DeploymentConfig using Restic/Kopia or restore hooks<a id="deployconfig"></a></h3>
 
--  (OADP 1.1+) **Error:** `DeploymentConfigs restore with spec.Replicas==0 or DC pods fail to restart if they crash if using Restic or restore hooks`
+-  (OADP 1.1+) **Error:** `DeploymentConfigs restore with spec.Replicas==0 or DC pods fail to restart if they crash if using Restic/Kopia or restore hooks`
```

Comment:
For OADP 1.1-1.2, it applies to DCs with restic restores or restore hooks. For OADP 1.3+, it applies to DCs with volumes (regardless of backup method) or restore hooks

Reply:
Is this the actual error from the OCP object itself ? I am asking, because if yes I need to match what's in the logs/error.

Reply:
It's not an "error" in the sense of a message/status. It's an error in the sense that behavior doesn't match expectations. For DCs with either volumes or hooks, our restore plugin disconnects pod from DC, scaling down DC and restoring bare pod. This means post-restore, application is running, data is restored, but they can't redeploy via DC. What the script does is delete these bare pods and scales DC back up to the original replicas value.

Reply:
I think that's sorted. I have changed this doc and added OADP 1.3+ which have reference to previous solution with different pre-condition.